### ISSUE TITLE
Server error on ECNF page

### ISSUE THREAD

**Issue by AndrewVSutherland** [LMFDB/lmfdb#2541]
Attempting to load the page

http://www.lmfdb.org/EllipticCurve/6.6.371293.1/1.1/a/2

takes a very long time and eventually results in a server error (tested on both prod and beta)

**Comment by edgarcosta** [comment]
all the time is being spent in a `pari_bnf` call which is being triggered by some line in  `make_E`

```
PATH: '/EllipticCurve/6.6.371293.1/1.1/a/2'
         2255643 function calls (2207428 primitive calls) in 73.389 seconds

   Ordered by: cumulative time, internal time, call count
   List reduced from 6643 to 30 due to restriction <30>

   ncalls  tottime  percall  cumtime  percall filename:lineno(function)
        1    0.000    0.000   73.397   73.397 /Applications/sage-dev/local/lib/python2.7/site-packages/werkzeug/contrib/profiler.py:95(runapp)
        1    0.000    0.000   73.397   73.397 /Applications/sage-dev/local/lib/python2.7/site-packages/flask/app.py:2262(wsgi_app)
        1    0.000    0.000   73.395   73.395 /Applications/sage-dev/local/lib/python2.7/site-packages/flask/app.py:1801(full_dispatch_request)
        1    0.000    0.000   73.391   73.391 /Applications/sage-dev/local/lib/python2.7/site-packages/flask/app.py:1779(dispatch_request)
        1    0.000    0.000   73.391   73.391 /Users/edgarcosta/Dropbox/projects/LMFDB/lmfdb/lmfdb/ecnf/main.py:334(show_ecnf)
        1    0.000    0.000   70.025   70.025 /Users/edgarcosta/Dropbox/projects/LMFDB/lmfdb/lmfdb/ecnf/WebEllipticCurve.py:243(by_label)
        1    0.000    0.000   69.905   69.905 /Users/edgarcosta/Dropbox/projects/LMFDB/lmfdb/lmfdb/ecnf/WebEllipticCurve.py:231(__init__)
        1    0.010    0.010   69.674   69.674 /Users/edgarcosta/Dropbox/projects/LMFDB/lmfdb/lmfdb/ecnf/WebEllipticCurve.py:253(make_E)
       11    0.003    0.000   60.185    5.471 /Users/edgarcosta/Dropbox/projects/LMFDB/lmfdb/lmfdb/utils.py:295(web_latex)
    66/11    0.000    0.000   60.184    5.471 /Applications/sage-dev/local/lib/python2.7/site-packages/sage/misc/latex.py:889(__call__)
        2    0.000    0.000   60.180   30.090 /Applications/sage-dev/local/lib/python2.7/site-packages/sage/rings/number_field/number_field_ideal.py:160(_latex_)
        2    0.000    0.000   60.178   30.089 /Applications/sage-dev/local/lib/python2.7/site-packages/sage/rings/number_field/number_field_ideal.py:469(_gens_repr)
        2    0.000    0.000   60.178   30.089 /Applications/sage-dev/local/lib/python2.7/site-packages/sage/rings/number_field/number_field_ideal.py:710(gens_reduced)
        1    0.001    0.001   60.178   60.178 /Applications/sage-dev/local/lib/python2.7/site-packages/sage/rings/number_field/number_field_ideal.py:1012(_cache_bnfisprincipal)
        1   60.174   60.174   60.174   60.174 /Applications/sage-dev/local/lib/python2.7/site-packages/sage/rings/number_field/number_field.py:3860(pari_bnf)
        1    0.008    0.008    7.983    7.983 /Users/edgarcosta/Dropbox/projects/LMFDB/lmfdb/lmfdb/ecnf/WebEllipticCurve.py:191(EC_nf_plot)
        6    0.003    0.001    7.856    1.309 /Users/edgarcosta/Dropbox/projects/LMFDB/lmfdb/lmfdb/ecnf/WebEllipticCurve.py:182(EC_R_plot)
    24/12    0.000    0.000    7.847    0.654 /Applications/sage-dev/local/lib/python2.7/site-packages/sage/misc/decorators.py:558(wrapper)
        6    0.000    0.000    7.830    1.305 /Applications/sage-dev/local/lib/python2.7/site-packages/sage/plot/contour_plot.py:872(implicit_plot)
     12/6    0.000    0.000    7.830    1.305 /Applications/sage-dev/local/lib/python2.7/site-packages/sage/misc/decorators.py:469(wrapper)
        6    0.807    0.134    7.829    1.305 /Applications/sage-dev/local/lib/python2.7/site-packages/sage/plot/contour_plot.py:227(contour_plot)
       18    0.001    0.000    6.895    0.383 /Applications/sage-dev/local/lib/python2.7/site-packages/sage/plot/misc.py:22(setup_for_eval_on_grid)
       18    0.003    0.000    6.894    0.383 {sage.ext.fast_eval.fast_float}
        1    0.004    0.004    6.862    6.862 /Applications/sage-dev/local/lib/python2.7/site-packages/sage/symbolic/expression_conversions.py:8(<module>)
        1    0.001    0.001    6.854    6.854 /Applications/sage-dev/local/lib/python2.7/site-packages/sage/symbolic/expression_conversions.py:687(__init__)
        1    0.001    0.001    5.652    5.652 /Applications/sage-dev/local/lib/python2.7/site-packages/sage/interfaces/sympy.py:31(<module>)
        1    0.004    0.004    5.651    5.651 /Applications/sage-dev/local/lib/python2.7/site-packages/sage/repl/ipython_extension.py:61(<module>)
        1    0.020    0.020    5.644    5.644 /Applications/sage-dev/local/lib/python2.7/site-packages/IPython/__init__.py:6(<module>)
        1    0.011    0.011    5.132    5.132 /Applications/sage-dev/local/lib/python2.7/site-packages/IPython/terminal/embed.py:4(<module>)
        1    0.012    0.012    4.245    4.245 /Applications/sage-dev/local/lib/python2.7/site-packages/IPython/terminal/interactiveshell.py:1(<module>)
```

**Comment by JohnCremona** [comment]
The comments in lines 365-368 of https://github.com/LMFDB/lmfdb/blob/master/lmfdb/ecnf/WebEllipticCurve.py
mention this curve specifically.  Working on it...

**Comment by AndrewVSutherland** [comment]
Closed via #2546.
